tensornetwork/backends/shell/shell_backend_test.py

```diff
@@ -1,134 +1,141 @@
-# Copyright 2019 The TensorNetwork Authors

```

Comment:
yeah, sorry, I have two git accounts and accidentally pushed/committed with the wrong one, so I had to alter the authorship post-hoc.

Reply:
Hmm, or maybe that wasn't it. 
did this solve it? Since I only added one function I just grabbed the master version and copy-pasted the multiply function from scalarouter.